test/classify.ts

```diff
@@ -89,12 +89,8 @@ describe("The classify endpoint", () => {
       ],
       inputs: ["brown"],
     });
-    expect(response.body.classifications[0].confidences).to.be.an("array");
-    expect(response.body.classifications[0].confidences[0]).to.have.property(
-      "option"
-    );
-    expect(response.body.classifications[0].confidences[0]).to.have.property(
-      "confidence"
+    expect(response.body.classifications[0]).to.not.have.property(
```

Comment:
I don't think this is really needed since the scope of change is on the lib level, so by default I don't have to assert that `xyz` does not exist in the response.

Reply:
I figured this is useful, since it's testing for a regression, but i think you're right in a sense that the SDK is not the best place to test for that.